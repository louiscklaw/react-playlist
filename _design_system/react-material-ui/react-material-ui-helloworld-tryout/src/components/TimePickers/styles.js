import { makeStyles } from '@material-ui/core/styles';

const LYNKED_LIGHT_GREY = '#AAAAAA';
const LYNKED_RED = '#800000';

const useStyles = makeStyles((theme) => ({
  container: {
    display: 'flex',
    flexWrap: 'wrap',
  },
  textField: {
    marginLeft: theme.spacing(1),
    marginRight: theme.spacing(1),
    // width: 200,
  },
  TextInput: {
    marginLeft: theme.spacing(1),
    marginRight: theme.spacing(1),

    // border: '1px solid #E0E0E0',

    padding: '3px 3px',
    '& ::placeholder': {
      color: LYNKED_LIGHT_GREY,
    },
  },
  error_message: {
    color: LYNKED_RED,

    display: 'flex',
    flexFlow: 'row',
    justifyContent: 'center',

    alignItems: 'center',
    paddingLeft: '0.25rem',
    paddingTop: '0.25rem',
    paddingBottom: '0.25rem',
  },
}));

export { useStyles };
