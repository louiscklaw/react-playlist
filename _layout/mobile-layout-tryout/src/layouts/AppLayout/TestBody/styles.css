.staggered-list-content {
  width: 400px;
  margin: 2rem auto;
}
.list {
  list-style-type: none;
  display: flex;
  flex-direction: column;
  height: 100vh;
  padding: 0;
}
.list li {
  width: 100%;
}
.list li + li {
  margin-top: 1rem;
}
.listItem {
  width: 100%;
  cursor: pointer;
  background-color: #d0d0d0;
  overflow: hidden;
}
.listItemContent {
  display: flex;
  flex-direction: row;
  align-items: center;
  padding: 1rem;
}
.avatar {
  width: 4rem;
  height: 4rem;
  border-radius: 100px;
  background-color: grey;
  margin-right: 2rem;
}
.avatarExpanded {
  width: 8rem;
  height: 8rem;
  margin-right: 0;
  margin-bottom: 1rem;
}
.description > div {
  background-color: grey;
  width: 14rem;
  border-radius: 6px;
  height: 0.5rem;
}
.description > div:nth-of-type(2) {
  width: 11rem;
}
.description > div:nth-of-type(3) {
  width: 8rem;
}
.description > div + div {
  margin-top: 1rem;
}
.expandedListItem .description {
  display: flex;
  align-items: center;
  flex-direction: column;
}
.expandedListItem {
  cursor: pointer;
  background-color: #d0d0d0;
}
.expandedListItemContent {
  padding: 2rem;
  display: flex;
  flex-direction: column;
  align-items: center;
}
.additional-content {
  width: 100%;
  margin-top: 2rem;
}

.additional-content > div {
  opacity: 0;
  border-radius: 3px;
  background-color: gray;
  height: 5rem;
}

/* content fade in animations */
@keyframes fadeIn {
  0% {
    opacity: 0;
    transform: translateY(20px);
  }
  100% {
    opacity: 1;
  }
}

.animated-in .additional-content > div {
  animation: fadeIn 0.4s forwards;
}

.additional-content > div:nth-of-type(2) {
  animation-delay: 0.15s;
}
.additional-content > div:nth-of-type(3) {
  animation-delay: 0.3s;
}
.additional-content > div + div {
  margin-top: 1rem;
}
