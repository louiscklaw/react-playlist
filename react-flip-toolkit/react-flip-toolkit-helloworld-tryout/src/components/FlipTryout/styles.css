* {
  box-sizing: border-box;
}

body {
  display: flex;
  justify-content: center;
  align-items: center;
  min-height: 100vh;
}

.square {
  transform-origin: 0 0;
  transform: rotateX(0);
  opacity: 1;
  width: 5rem;
  height: 5rem;
  cursor: pointer;
  background-image: linear-gradient(
    45deg,
    rgb(121, 113, 234),
    rgb(97, 71, 182)
  );
}

.full-screen-square_a {
  transform-origin: 0 0;
  transform: rotateX(-70deg);
  opacity: 0.5;
  width: 5rem;
  height: 5rem;
  cursor: pointer;
  background-color: red;
}
