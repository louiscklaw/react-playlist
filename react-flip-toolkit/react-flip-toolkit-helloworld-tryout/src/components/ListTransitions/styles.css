body {
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
}
.fm-example {
  padding: 2rem;
  min-height: 150vh;
}

.fm-example > * {
  max-width: 1100px;
  margin-left: auto;
  margin-right: auto;
}

.fm-grid > ul {
  list-style-type: none;
  display: grid;
  grid-template-columns: repeat(auto-fill, minmax(14rem, 1fr));
  grid-auto-rows: 8rem;
  grid-gap: 1rem;
  padding: 0;
  margin: 0;
}

.fm-list {
  grid-gap: 1rem;
  margin: 2rem 0;
}

.fm-grid,
.fm-list {
  border-radius: 5px;
  background-color: #ececec;
  margin-top: 2.5rem;
  max-width: 900px;
}

.fm-list > ul,
.fm-grid > ul {
  width: 100%;
  list-style-type: none;
  margin: 0;
  padding: 1rem;
}

.fm-list .fm-item {
  margin-bottom: 1rem;
}

.fm-list .fm-item:last-of-type {
  margin-bottom: 0;
}

.fm-item {
  overflow: hidden;
  display: block;
  border-radius: 4px;
  background-color: white;
}

.fm-item h3 {
  font-size: 1.1rem;
}

.fm-item > div {
  padding: 0 1.25rem;
  display: flex;
  justify-content: space-between;
  align-items: flex-start;
}

.fm-item h3 {
  margin-bottom: 0;
}

.fm-item p {
  font-style: italic;
  margin-top: 0;
}

.fm-remove {
  border: 0;
  background: transparent;
  cursor: pointer;
  font-size: 2.2rem;
  color: #ff4057;
  position: relative;
  left: 0.5rem;
  width: 1.5rem;
}

.fm-show-all {
  padding: 0.5rem 1rem;
  background-color: #ff4057;
  color: white;
  font-weight: bold;
  border: 0;
  border-radius: 3px;
}

.fm-flex-container {
  display: flex;
  flex-wrap: wrap;
}

.fm-flex-container > label {
  min-width: 4.5rem;
}

legend {
  font-weight: bold;
}

.fm-example fieldset {
  border: 0;
  padding: 1rem;
}

.fm-example label {
  margin-right: 1.5rem;
  cursor: pointer;
}

.fm-example input {
  position: relative;
  margin-right: 0.5rem;
}
