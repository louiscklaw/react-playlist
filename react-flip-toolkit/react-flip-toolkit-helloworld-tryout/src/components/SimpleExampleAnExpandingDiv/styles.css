* {
  box-sizing: border-box;
}

body {
  display: flex;
  justify-content: center;
  align-items: center;
  min-height: 100vh;
}

.square {
  width: 5rem;
  height: 5rem;
  cursor: pointer;
  background-image: linear-gradient(
    45deg,
    rgb(121, 113, 234),
    rgb(97, 71, 182)
  );
}

.full-screen-square1 {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  cursor: pointer;
  background-image: linear-gradient(
    45deg,
    rgb(121, 113, 234),
    rgb(97, 71, 182)
  );
}
