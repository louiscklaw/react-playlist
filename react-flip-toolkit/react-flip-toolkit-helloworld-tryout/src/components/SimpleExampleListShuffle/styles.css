* {
  box-sizing: border-box;
}

body {
  font-size: 2.5rem;
  font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI",
    "Roboto", "Helvetica Neue", Arial, sans-serif;
}

.list {
  list-style-type: none;
  display: flex;
}

.list > li {
  margin-right: 1rem;
}
