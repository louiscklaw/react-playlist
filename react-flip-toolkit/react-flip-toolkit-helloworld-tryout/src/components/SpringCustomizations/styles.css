* {
  box-sizing: border-box;
}

@import url("https://fonts.googleapis.com/css?family=Source+Code+Pro:400");

body {
  padding: 2rem 1rem;
  -webkit-font-rendering: antialiased;
  font-size: 15px;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
}

.container {
  max-width: 800px;
  margin: auto;
}

label {
  display: block;
}

.container > div:first-of-type {
  display: flex;
}

.container > div:first-of-type > * {
  flex: 1;
}
.item {
  height: 2.5rem;
  margin-bottom: 1rem;
  cursor: pointer;
  position: relative;
  box-shadow: 0 4px 6px hsla(0, 0%, 0%, 0.2);
}

input {
  margin-right: 0.5rem;
}

label {
  margin-right: 1.5rem;
  min-height: 1.75rem;
}

legend {
  font-weight: bold;
  margin-bottom: 0.5rem;
}

input[type="radio"]:first-of-type {
  margin-left: 0;
}

.main {
  margin: 2rem 0 2rem 0;
}

fieldset {
  border: 0;
  margin-top: 1rem;
  display: inline-block;
}

input[type="number"] {
  width: 3.5rem;
  margin-left: 0.5rem;
}
