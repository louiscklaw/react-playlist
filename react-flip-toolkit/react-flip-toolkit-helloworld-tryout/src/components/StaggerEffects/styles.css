.staggered-list-content {
  width: 400px;
  margin: 2rem auto;
}
.list {
  list-style-type: none;
  display: flex;
  flex-direction: column;
  height: 100vh;
  padding: 0;
}
.list li {
  width: 100%;
}
.list li + li {
  margin-top: 10px;
}
.listItem {
  width: 100%;
  cursor: pointer;
  background-color: #d0d0d0;
  overflow: hidden;
}
.listItemContent {
  display: flex;
  flex-direction: row;
  align-items: center;
  padding: 1rem;
}
.avatar {
  width: 5rem;
  height: 5rem;
  border-radius: 10px;
  margin-right: 2rem;

  background-color: grey;
}
.avatarExpanded {
  width: 15rem;
  height: 15rem;
  margin-right: 0;
  margin-bottom: 1rem;
}
.description > div {
  /* background-color: grey; */
  width: 14rem;
  border-radius: 6px;
  height: 0.5rem;
  font-size: 0.8rem;
}
.description > div:nth-of-type(2) {
  width: 11rem;
}
.description > div:nth-of-type(3) {
  width: 8rem;
}
.description > div + div {
  margin-top: 1rem;
}
.expandedListItem .description {
  display: flex;
  align-items: center;
  flex-direction: column;
}
.expandedListItem {
  cursor: pointer;
  background-color: #d0d0d0;
}
.expandedListItemContent {
  padding: 2rem;
  display: flex;
  flex-direction: column;
  align-items: center;
}
.additional-content {
  width: 100%;
  margin-top: 2rem;
}

.additional-content > div {
  opacity: 0;
  border-radius: 3px;
  /* background-color: gray; */
  /* height: 5rem; */
}

/* content fade in animations */
@keyframes fadeIn {
  0% {
    opacity: 0;
    transform: translateY(20px);
  }
  100% {
    opacity: 1;
  }
}

.animated-in .additional-content > div {
  animation: fadeIn 0.4s forwards;
}

.additional-content > div:nth-of-type(2) {
  animation-delay: 0.15s;
}
.additional-content > div:nth-of-type(3) {
  animation-delay: 0.3s;
}
.additional-content > div + div {
  margin-top: 1rem;
}
