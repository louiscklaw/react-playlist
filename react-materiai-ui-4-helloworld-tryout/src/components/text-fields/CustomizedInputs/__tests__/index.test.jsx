import { createShallow, createRender, createMount } from '@material-ui/core/test-utils';
import { ThemeProvider } from '@material-ui/core/styles';
import { MockedTheme } from 'src/tests/MockedTheme';
import CustomizedInputs from 'src/components/text-fields/CustomizedInputs/index.jsx';

describe('<CustomizedInputs> helloworld', () => {
  describe('<CustomizedInputs /> mount helloworld', () => {
    let mount;

    beforeEach(() => {
      mount = createMount();
    });

    afterEach(() => {
      mount.cleanUp();
    });

    test('should work', () => {
      const wrapper = mount(
        <MockedTheme>
          <CustomizedInputs>CustomizedInputs</CustomizedInputs>
        </MockedTheme>
      );

      expect(wrapper.html).toMatchSnapshot();
    });
  });

  describe('<CustomizedInputs /> render helloworld', () => {
    let render;

    beforeEach(() => {
      render = createRender();
    });

    afterEach(() => {});

    test('should work', () => {
      const wrapper = render(
        <MockedTheme>
          <CustomizedInputs>CustomizedInputs</CustomizedInputs>
        </MockedTheme>
      );

      expect(wrapper.html).toMatchSnapshot();
    });
  });

  describe('<CustomizedInputs /> shallow helloworld', () => {
    let shallow;

    beforeEach(() => {
      // This is Mocha; in Jest, use beforeAll
      shallow = createShallow();
    });

    test('should work', () => {
      const wrapper = shallow(<CustomizedInputs>CustomizedInputs</CustomizedInputs>);

      expect(wrapper.html).toMatchSnapshot();
    });
  });
})
