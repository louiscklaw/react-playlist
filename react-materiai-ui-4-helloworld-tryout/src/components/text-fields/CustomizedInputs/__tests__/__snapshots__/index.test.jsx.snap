// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomizedInputs> helloworld <CustomizedInputs /> mount helloworld should work 1`] = `[Function]`;

exports[`<CustomizedInputs> helloworld <CustomizedInputs /> render helloworld should work 1`] = `[Function]`;

exports[`<CustomizedInputs> helloworld <CustomizedInputs /> shallow helloworld should work 1`] = `[Function]`;
